lang/main.json

```diff
@@ -450,7 +450,12 @@
         "on": "Recording",
         "pending": "Preparing to record the meeting...",
         "rec": "REC",
+        "authDropboxText": "To start the recording you need to first authorize our Dropbox recording app. After the recording is finished the file will be uploaded to your Dropbox account.",
```

Comment:
I would just replace this by "Upload your recording to Dropbox." and then add a "sign in" text to the button. I think the other part is kind of obvious.